src/Model/Catalog/CfdiType.php

```diff
@@ -184,14 +184,14 @@ public static function findByName($name)
      *
      * @throws InvalidArgumentException
      *
-     * @return array The matching `CFDI_TYPE_*` constant for the given CFDI type value
+     * @return \Generator The matching `CFDI_TYPE_*` constants for the given CFDI type value
```

Comment:
Replace docblock by `{@inheritdoc}`.

Reply:
This is not possible since here we are overriding the original docblock from the interface.

Reply:
The same situation is happening in the other catalogues. You should update all. I prefer use `{@inheritdoc}` anyways.